EngineLayer/Psm.cs

```diff
@@ -401,14 +401,14 @@ internal void Add(Psm psmParent)
         private Tuple<string, double?> Resolve(IEnumerable<double> enumerable)
         {
             double doubleTol = 1e-6;
-            var list = enumerable.ToList();
+            var list = enumerable.Distinct().ToList();
```

Comment:
there are cases when the PSM repeats the same base sequence and the same protein. like so:


MFGMIK or MFGMIK or MGFMIK or MGFMIK DECOY_P04803  DECOY_P04803 Q06834  Q06834